### ISSUE TITLE
Document each API route with an example curl

### ISSUE THREAD

**Issue by mattbucci** [vervallsweg/cast-web-api#27]
It'd be really helpful if you could provide a single line command to test some of these commands out.

I'm currently trying `curl http://10.0.0.139:3000/setMediaPlayback\?address\=10.0.0.126\&mediaType\=audio/mp3\&mediaUrl\=https://archive.org/download/testmp3testfile/mpthreetest.mp3\&mediaStreamType\=BUFFERED\&mediaTitle\="test"\&mediaSubtitle\="test"\&mediaImageURL\="test.png"` and having no luck, I'll go dig through the source code to figure out what I'm doing wrong but as both the app and the docs don't give you a good indication of what these values should be I'm not sure :)

**Comment by vervallsweg** [comment]
Hey, so does the last command work for you now? Tried running it from Chrome and curl, both work for me.

Yes adding an example URL to each command is something I could do in the future. But it probably wouldn't mention curl specifically. I mean you'd just have to paste the example into your curl command: curl "future example url".

If that's something that would make the docs easier to understand let me know and I'll do it :)

**Comment by mattbucci** [comment]
Yeah the second url managed to work but yeah adding an example URL you can paste would be super useful I forgot that none of them are post requests

**Comment by vervallsweg** [comment]
Ok, added an example URL for each API route. Will soon merge v0.2 into master. Should the API become more complex, I'll add swagger-ui to it.
